brave/brave-ios
# Fix 2939: Reader mode should work for all URLs

## Summary of Changes


This pull request fixes https://github.com/brave/brave-ios/issues/2939

## Submitter Checklist:


## Test Plan:

Specified here: https://github.com/brave/brave-ios/issues/2939

1. Navigate to https://www.cnn.com/2020/10/07/weather/hurricane-delta-wednesday/index.html - confirm reader mode works
2. Transition to a different app (without closing the app) and open brave again
3. Verify that reader mode still works.

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
